src/index.html

```diff
@@ -15,8 +15,91 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <!-- <h1>Moyo header</h1> -->
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute of the `img` tag should describe the image content. For example, if it's a logo of a company, you can write the company's name. (HTML Content)